### ISSUE TITLE
H3: mozilla.org/contribute: A/B testing live + early results analysis

### ISSUE THREAD

**Issue by LucyeoH-zz** [mozilla/participation-org#37]
<h1>Project Description</h1>

Owner: @Lucyeoh
Members: @ppapadeas, @habber, Jen Bertsch, @craigcook
Informed: Mary Ellen, George

Designing a simplified interface specifically tailored towards people new to Mozilla, who want to contribute in some form (not donations) but not sure what they could be doing. For more project background see #9 

<h1>Primary Goals & Objectives for This Heartbeat </h1>
- [x] Get project management transferred from @djst to @lucyeoh 
- [x] Launch A/B testing site (June 18th) 
- [x] Create approach for compiling and analyzing early results 

<h1>Milestones</h1>
- [x] Kickoff Meeting
- [x] Transfer meeting with Lucy and David
- [x] Meet with Nikos and Pierros to get update about development timeline ( @comzeradd @ppapadeas Friday June 5th) 
- [x] Set analytics meeting to figure out how we'll measure - Craig Cook and Mike Alexis, Adam Lofting ,  Jen Bretch + Nikos and Pierros (Tuesday June 9th)
- [x] Finish all copy (@lucyeoh due Tuesday May 3rd) https://etherpad.mozilla.org/contibutepagecopy
- [x] Create new task to replace file your first bug (@lucyeoh Due Monday June 8)
- [x] Analytics Meeting - Metrics issues decided  (Tuesday June 9th)  
- [x] Make sure Bobby and team have proper landing page (Due Wed June 9, @secretrobotron) 
- [x] Create survey for ratings page (touch base with Craig/Nikos) (Due Wed June 9 @habber)
- [x] Set up Optimizely on the A/B testing page (Adam Lofting as a resource) (Due Friday June 12th @comzeradd) 
- [x] Discuss collecting & responding to emails with Jen Bertsch, Jen Neely, Jess Davis - Email flow drip campaign ( @lucyeoh Due Thursday June 11th) 
- [x] Build confirmation page for A/B Tests ( @comzeradd due Friday June 12th) 
- [x] All code done ( @comzeradd due Saturday June 13th) 
- [x] Confirmation page for control page + code review  ( @craigcook due Wed June 17) 
- [x] Site up and running ((If tests pass) Thursday June 18th) 
- [x] Wrap-up Meeting Friday June 19th

Next Heartbeat: 
- Adam to help with preliminary analysis of Optomizely results

**Comment by LucyeoH-zz** [comment]
@comzeradd @ppapadeas @craigcook any thoughts about how we can link up the "confirmation page" with the star ratings and if we can customize the CTA at the bottom of the page based on the task they just completed? Details here: https://bugzilla.mozilla.org/show_bug.cgi?id=1171670

**Comment by LucyeoH-zz** [comment]
@nukeador reminder that we're switching out "file a bug" task with "test a mobile app". All of the copy is here - https://etherpad.mozilla.org/contibutepagecopy

**Comment by secretrobotron** [comment]
> Make sure Bobby and team have proper landing page

This is what I've got for now: http://mozilla.github.io/webmaker-launch/activity.html

When we launch on the play store, I'll update that. We'll have some better stuff leading up to Whistler too, and proper landing page for the whole platform (e.g. beta.webmaker.org).

**Comment by habber** [comment]
Regarding the star rating, assuming we can track the issue chosen in the prior step, would it be possible to create a rating question in Survey Gizmo for each issue and then display the survey question relevant to the user?

I can create the Survey Gizmo survey questions if it sounds like a direction we want to take for this in order to track/capture feedback.

**Comment by comzeradd** [comment]
~~I'll be opening the PR today. I've already [pushed the code](https://github.com/comzeradd/bedrock/commit/b1d6e69b0dd6f0a3c279cb5163bd46140bfb84b5), but it still needs some minor fixes.~~

[PR opened](https://github.com/mozilla/bedrock/pull/3059)

**Comment by brianking** [comment]
@LucyeoH  Please edit this comment and fill out this template. Thanks!

### What was supposed to happen?

I took over the A/B test from David Tenser start of June, we were supposed to finish the coordination and the test was supposed to be up and running by the end of this quarter. 
...

### What did happen?

At first everything was running very smoothly (as you can see above) but then just prior to Whistler we became blocked by the need to create a new Exact Target bucket to collect people in. I got in touch with Jessilyn got keys to Exact Target and we resolved it right after Whistler. 

However we then got blocked by a last review where it came came up that some extra work was required on the backend that hadn't been taken into consideration in the beginning (mostly writing tests). Now waiting on reviews [from a few people](https://github.com/mozilla/bedrock/pull/3059) and on a merge.  

Bug details [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1168359#c6). 

...

### What we learned?

Code projects are large and complicated beasts and it's difficult to manage without understanding the technical side. Harder to figure out whose waiting on what for who and what stage people are at. 

Also different groups having conversations in multiple places (GitHub + Bugzilla) made it harder to follow the progress of the project. 
...

### What are some improvements?

More regular vidyo meetings to make sure everyone is on the same page will hopefully lead to clearer and more regular communication about blockers and bugs so that they can be resolved more quickly. 
...

### What does the next Heartbeat for this project look like?

Reviewing the data and using it to craft a strategy for the site moving forward. 
While also having lots of discussions with stakeholders and strategy conversations about what this page needs to be to provide advantage. Needs also to fit in with other participation initiatives. Hopefully start crafting a clear plan for this page moving forward. 
...

**Comment by brianking** [comment]
Hi ... just a reminder about this one @LucyeoH

**Comment by LucyeoH-zz** [comment]
@brianking updated!

**Comment by comzeradd** [comment]
This is live now. [Bug 1168359](https://bugzilla.mozilla.org/show_bug.cgi?id=1168359)
